stdlib/public/core/AssertCommon.swift

```diff
@@ -29,8 +29,9 @@ func _isDebugAssertConfiguration() -> Bool {
   return Int32(Builtin.assert_configuration()) == 0
 }
 
-@usableFromInline @_transparent
-internal func _isReleaseAssertConfiguration() -> Bool {
+@_transparent
+public // @testable, used in _Concurrency executor preconditions
+func _isReleaseAssertConfiguration() -> Bool {
```

Comment:
All other methods here are exposed like this, including `_isDebugAssertConfiguration` and friends, and we'd like to use this one as well in order to implement a precondition in Concurrency that efficiently checks executor, so I made this follow the same public + transparent pattern as other methods here.

Any objections to that? @DougGregor @mikeash @kavon?

Reply:
Yeah they're pretty boring, and all the same shape, I don't expect trouble in exposing this one 🤔